spec/basic/lib_spec.js

```diff
@@ -108,6 +108,11 @@ describe('protractor library', function() {
         toEqual('repeater');
   });
 
+  it('should allow self-wrapped webdriver instances', function() {
+    var driver = protractor.wrapDriver(browser.driver);
+    driver.get('https://angularjs.org/');
```

Comment:
Probably you need to add some expectations here. As in future team members won't know if you intentionally missed `expect` or you just forgot.